### PR TITLE
doc: minimum volume size supported depends upon the underlying brick'…

### PR DESCRIPTION
**Pull request by SaravanaStorageNetwork**
doc: minimum volume size supported depends upon the underlying brick's size

Signed-off-by: Saravanakumar Arumugam <sarumuga@redhat.com>

### CODE REVIEW

File: doc/admin/volume.md [hunk 1]
```diff
@@ -57,6 +57,13 @@ OPTIONS
 	snapshot storage, then snapshot-factor would be set to 1.5.  If the
 	value is set to 1, then snapshots will not be enabled for this volume (default 1)
 
+Note:
+The volume size created depends upon the underlying brick size.
+For example, for a 2 way/3 way replica volume, the minimum volume size is 1GiB as the
+underlying minimum brick size is constrained to 1GiB.
+
+So, it is not possible create a volume of size lesser than 1GiB.
```
Comment (obnoxxx) [inline doc/admin/volume.md]: less than

File: doc/api/api.md [hunk 2]
```diff
@@ -511,6 +511,12 @@ These APIs inform Heketi to create a network file system of a certain size avail
     ]
 }
 ```
+Note:
+The volume size created depends upon the underlying brick size.
+For example, for a 2 way/3 way replica volume, the minimum volume size is 1GiB as the
+underlying minimum brick size is constrained to 1GiB.
+
+So, it is not possible create a volume of size lesser than 1GiB.
```
Comment (obnoxxx) [inline doc/api/api.md]: less than

File: doc/man/heketi-cli.8 [hunk 3]
```diff
@@ -416,6 +416,14 @@ For example if a 10TiB volume requires 5TiB of snapshot storage, then snapshot\-
 If the value is set to 1, then snapshots will not be enabled for this volume.
 .RE
 .PP
+\fBNote:\fP
+The volume size created depends upon the underlying brick size.
+For example, for a 2 way/3 way replica volume, the minimum volume size is 1GiB as the
+underlying minimum brick size is constrained to 1GiB.
+So, it is not possible create a volume of size lesser than 1GiB.
```
Comment (obnoxxx) [inline doc/man/heketi-cli.8]: less than

File: doc/admin/volume.md (hunk 1 above)
Comment (SaravanaStorageNetwork) [reply doc/admin/volume.md]: done

File: doc/api/api.md (hunk 2 above)
Comment (SaravanaStorageNetwork) [reply doc/api/api.md]: done

File: doc/man/heketi-cli.8 (hunk 3 above)
Comment (SaravanaStorageNetwork) [reply doc/man/heketi-cli.8]: done

**Pull request merged**